### PR TITLE
fix(deploy): check client certificate exists before creating deployment

### LINKED ISSUE #1164: `akash deploy` does not require/create a client certificate
Like `akash tx deployment create` does.

### PR DESCRIPTION
**Pull request by troian**
fixes #1164

Signed-off-by: Artur Troian <troian.ap@gmail.com>

### CODE REVIEW

File: deploy/cmd/create.go [hunk 1]
```diff
@@ -75,6 +76,15 @@ func createCmd() *cobra.Command {
 				return err
 			}
 
+			if _, err = cutils.LoadCertificateForAccount(clientCtx, clientCtx.Keyring); err != nil {
+				if os.IsNotExist(err) {
```
Comment (hydrogen18) [inline deploy/cmd/create.go:80]: Shouldn't we also check that the certificate is on chain?

Comment (troian) [reply deploy/cmd/create.go:80]: 💯

Comment (troian) [reply deploy/cmd/create.go:80]: fixed

**Review by boz — APPROVED** [review]
:rocket:

**Pull request merged**